sql/core/src/test/scala/org/apache/spark/sql/QueryTest.scala

```diff
@@ -235,7 +235,21 @@ abstract class QueryTest extends PlanTest {
   }
 }
 
-object QueryTest {
+object QueryTest extends Assertions {
+  /**
+   * Runs the plan and makes sure the answer matches the expected result.
+   *
+   * @param df the [[DataFrame]] to be executed
```

Comment:
Ah ... the problem was this ... 

```
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/QueryTest.java:8: error: reference not found
[error]    * match the expected result, an error message will be returned. Otherwise, a {@link None} will
[error]                                                                                        ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/QueryTest.java:11: error: reference not found
[error]    * @param df the {@link DataFrame} to be executed
[error]                           ^
[error] /home/jenkins/workspace/SparkPullRequestBuilder/sql/core/target/java/org/apache/spark/sql/QueryTest.java:12: error: reference not found
[error]    * @param expectedAnswer the expected result in a {@link Seq} of {@link Row}s.
[error]          
```

Reply:
`DataFrame` is `Dataset<Row>` in Java API .. and type alias doesn't work with Unidoc when it's linked.. you can switch to `DataFrame` .. :-).

Some error messages are spurious from `javadoc` but it's difficult to fix. See https://issues.apache.org/jira/browse/SPARK-20840